### PR TITLE
added 🇮🇩 Indonesia (id-ID)  in locales i18n

### PR DESCRIPTION
**Pull request by ri7nz**
Hi @wojtekmaj , Feel free to review my PRS. 
image
image

### CODE REVIEW

**Comment by wojtekmaj** [comment]
Hi,
thanks for the PR! You're missing a few strings. Would you be so kind and add the missing ones? Fully translated is e.g. upside down Indonesia 🇵🇱 :) https://github.com/wojtekmaj/react-lifecycle-methods-diagram/blob/master/src/i18n/json/pl-PL.json

**Comment by ri7nz** [comment]
@wojtekmaj Done Sir 👍

**Comment by wojtekmaj** [comment]
Still missing "React version" and "Language" :)

**Comment by ri7nz** [comment]
@wojtekmaj Oh, I'm sorry for my less thorough 🙏 . you can review again 👍

**Comment by wojtekmaj** [comment]
@ri7nz I think "Language" is not "Indonesia" in Indonesian :) Fixed that for you, but need your confirmation if it's alright. If that's fine, Let's merge this 💪

File: src/i18n/json/id-ID.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "Options": "Opsi",
   "Show less common lifecycles": "Tunjukkan lifecycles yang kurang umum",
   "React version": "Versi React",
-  "Language": "Indonesia",
+  "Language": "Bahasa",
```
Comment (ri7nz) [inline src/i18n/json/id-ID.json]: Its okay 👍 @wojtekmaj

**Comment by wojtekmaj** [comment]
Thank you so much!

**Pull request merged**